native/cocos/ui/videoplayer/VideoPlayer-java.cpp

```diff
@@ -43,9 +43,11 @@ using namespace cc; //NOLINT
 static void executeVideoCallback(int index, int event);
 
     #define QUIT_FULLSCREEN 1000
+    #define JNI_EXPORT __attribute__((visibility("default")))
 
 extern "C" {
 //NOLINTNEXTLINE
+JNI_EXPORT
 void Java_com_cocos_lib_CocosVideoHelper_nativeExecuteVideoCallback(JNIEnv *env, jobject obj, jint index, jint event) {
```

Comment:
Don't have to define it manually. Can refer to the usage in `platform/android/jni/JniCocosEntry.cpp`.